src/msw/webview_ie.cpp

```diff
@@ -923,6 +923,11 @@ bool wxWebViewIE::RunScript(const wxString& javascript, wxString* output)
                     dispatchResult->AddRef();
                     if ( JSONAO.Invoke("stringify", DISPATCH_METHOD, varJSONStr, 1, &varResult) )
                         varResult = varJSONStr;
+                    else
+		    {
+		        wxLogWarning(_("JSON.stringify fails when trying to convert object into JSON"));
+                        return false;
```

Comment:
Absolutely, I missed that. Well, I don't know about if I introduced tabs. I will check that again because I commonly program with tabs, so maybe I add some by mistake.